tbx/project_styleguide/templates/patterns/organisms/footer/footer.html

```diff
@@ -4,7 +4,11 @@
 
     <div class="grid footer__top ">
         <span class="grid__footer-line footer__line"></span>
-        {% include "patterns/molecules/footer-cta/footer-cta.html"%}
+        {% with contact=page.footer_contact %}
+            {% if contact %}
+                {% include "patterns/molecules/footer-cta/footer-cta.html" with contact_heading=contact.title contact_text=contact.text contact_link=contact.link contact_name=contact.name contact_role=contact.role contact_image=contact.image contact_action=contact.button_text %}
```

Comment:
I don't think the footer-cta template will be re-used, so it would be fine to update the values in that file directly if you would prefer to.